html/semantics/interactive-elements/the-dialog-element/abspos-dialog-layout.html

```diff
@@ -55,6 +56,8 @@
 
 function reset() {
     document.body.style.width = "auto";
+    dialog.style.top = null;
+    dialog.style.height = null;
```

Comment:
Would we use `unset` better?

Reply:
`dialog.style.height = 'unset'` would result in `<dialog style="height: unset"></dialog>`

`dialog.style.height = null` results in `<dialog style=""></dialog>`

Reply:
Yeah, this is the right thing to do (or just `= "";`, or `.removeProperty("height")`.